Mathlib/Topology/Instances/AddCircle.lean

```diff
@@ -137,6 +137,9 @@ theorem coe_sub (x y : 𝕜) : (↑(x - y) : AddCircle p) = (x : AddCircle p) -
 theorem coe_neg {x : 𝕜} : (↑(-x) : AddCircle p) = -(x : AddCircle p) :=
   rfl
 
+theorem coe_zero : ↑(0 : 𝕜) = (0 : AddCircle p) :=
```

Comment:
```suggestion
theorem coe_zero : (0 : 𝕜) = (0 : AddCircle p) :=
```
It should be exactly the same.

Reply:
Whoops! I should have thought of this the second time around 😅

Reply:
The danger with it being written without any arrows is that someone could introduce a (truncating!) coercion in the other direction, and then you'd have no clue what the statement actually is!

Reply:
I'll revert the change, then!